### ISSUE TITLE
[BUG] Sisimai.make() freezes on FreeBSD

### ISSUE THREAD

**Issue by azumakuniyuki** [sisimai/rb-Sisimai#56]
`Sisimai.make()` and `dump()` method freeze on FreeBSD when the first argument is a path to a directory.

```shell
% uname -srpm
FreeBSD 10.1-RELEASE-p24 amd64 amd64
% ruby -v
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-freebsd10.1]

% ruby -rsisimai -e 'print Sisimai.make($*.shift).size' ./set-of-emails/maildir/bsd
^C/usr/local/lib/ruby/gems/2.2.0/gems/sisimai-4.17.0/lib/sisimai/message.rb:46:in `initialize': Interrupt
```

Something wrong in Sisimai::Mail::Maildir class, perhaps.

**Comment by azumakuniyuki** [comment]
The patch provided from Pull-Request #57 has resolved this issue !

```shell
% cd /usr/local/src/rb-Sisimai/
% git pull
remote: Counting objects: 16, done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 16 (delta 2), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (16/16), done.
From https://github.com/sisimai/rb-Sisimai
   2c01fcd..ff39f07  master     -> origin/master
Updating 2c01fcd..ff39f07
Fast-forward
 lib/sisimai/mail/maildir.rb       |  7 +++++--
 spec/sisimai/mail/maildir_spec.rb | 32 +++++++++++++++++++-------------
 spec/sisimai/mail_spec.rb         | 19 ++++++++++---------
 3 files changed, 34 insertions(+), 24 deletions(-)

% ruby -Ilib -rsisimai -e 'print Sisimai.make($*.shift).size' ./set-of-emails/maildir/bsd
365
```

**Comment by azumakuniyuki** [comment]
Resolved at Sisimai v4.17.1
